torchtune/models/llama3/_tokenizer.py

```diff
@@ -241,13 +241,29 @@ def tokenize_messages(
         self,
         messages: List[Message],
         add_eos: bool = True,
+        add_eot: bool = True,
     ) -> Tuple[List[int], List[bool]]:
         """
         Tokenize a list of messages into a list of token ids and masks.
 
         Args:
             messages (List[Message]): The list of messages to tokenize.
-            add_eos (bool): Wether to add the tokenizer's eos_id. Default True.
+            add_eos (bool): Add the tokenizer's ``eos_id`` (end-of-sequence ID). Default True.
+            add_eot (bool): Add the tokenizer's ``eot_id`` (end-of-turn ID) to final assistant message. Default True.
```

Comment:
would be good to explain what scenarios to use this parameter in (i.e., for generation you would set this to False, in most other cases it should be True)

Reply:
Excellent point.